packages/miniconda3.rb

```diff
@@ -3,33 +3,20 @@
 class Miniconda3 < Package
   description 'Anaconda is the world\'s most popular Python data science platform.'
   homepage 'https://conda.io/miniconda.html'
-  version '5.0.1'
+  version '4.4.10'
```

Comment:
Downgrade?

Reply:
No I messed up originally and used the version numbers from Anaconda instead of Miniconda because I'm not all there in the head.